app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala

```diff
@@ -19,10 +19,26 @@ package uk.gov.hmrc.bindingtariffclassification.model
 import java.time.{Instant, ZoneId, ZonedDateTime}
 
 import play.api.libs.json._
+import uk.gov.hmrc.crypto.{CompositeSymmetricCrypto, Crypted, PlainText}
 import uk.gov.hmrc.play.json.Union
 
 object MongoFormatters {
 
+  private val crypto = CompositeSymmetricCrypto.aes("MZygpewJsCpRrfOrr77j53", Seq.empty) // TODO Pull the real key from config
```

Comment:
Not sure the best way to pull this from config given MongoFormatters is an Object

Reply:
You can read the config in a new trait or singleton class you create. 
Your object can extend it.